apps/www/_blog/2024-12-06-database-build-v2.mdx

```diff
@@ -0,0 +1,218 @@
+
+For a behind-the-scenes look at the engineering behind this, check out our dedicated [blog post](/blog/database-build-live-share).
+
+## Deployments
+
+One of the most requested features has been a way to easily deploy your databases to the cloud with a single click.
+
+As a quick reminder - all database.build databases run directly in your browser using PGlite. This client-side approach makes it easy to spin up virtually limitless databases for design and experimentation. But when it’s time to use your database in production, the options have been somewhat cumbersome:
+
+- Manually copy-pasting the generated migrations and running them against a real database
+- Using Live Share with `pg_dump` and `pg_restore` to back up and restore your browser database to a production-ready database
+
+Now with deployments, you can deploy your database.build creations directly to a cloud database provider, starting with Supabase (and AWS coming soon).
+
+<video width="99%" autoPlay loop muted playsInline controls={true}>
+  <source
+    src="/images/blog/launch-week-13/day-5-database-build-v2/deploy-to-supabase.mp4"
+    type="video/mp4"
+  />
+</video>
+
+### Deploy to Supabase
+
+After building your database schema, click the **Deploy** button in the app’s header.
+
+The first time you deploy to Supabase, you will need to connect your database.build account with your Supabase account. If you don’t already have a Supabase account, you’ll be given the option to create a new one here. Supabase includes a Free plan, so if this is your first project there will be no cost to deploy. Follow the steps outlined in the dialog to connect your account with a Supabase org.
+
+Once your account is connected you will be presented with a preview of the org and project name that will be created. If you are happy with this, click **Deploy**. Be sure to keep your browser tab open to ensure a successful deployment.
+
+Finally you will be presented with connection strings and a password to connect to your new database.
+
+### Behind the scenes
+
+Getting deployments to work took quite a bit more engineering than it might seem at first glance. To copy your in-browser database to the cloud accurately, we needed a way to dump and restore. ElectricSQL has been hard at work on a WASM version of `pg_dump` that can run directly in the browser—a mind-blowing feat in its own right—but unfortunately, it wasn’t quite ready in time for this launch.
+
+In the meantime, we decided to piggyback on top of our existing Live Share feature, which creates a reverse connection between your in-browser database and an external PostgreSQL client. With Live Share enabled, we connect a server-side `pg_dump` program directly to your in-browser database. The resulting dump is then piped into `pg_restore`, which transfers it to your new cloud database.
+
+![database.build v2 Deploy to Supabase](/images/blog/launch-week-13/day-5-database-build-v2/deploy-to-supabase-dark.png)
```

Comment:
@fsansalvadore i couldn't get this diagram right. need to make sure it looks good in both dark and light modes.